src/components/Blocks/Banner/Banner.tsx

```diff
@@ -12,6 +12,7 @@ import { Icon28Close } from 'icons/28/close';
 import { Icon28CloseAmbient } from 'icons/28/close_ambient';
 
 import { Tappable } from 'components/Service/Tappable/Tappable';
+import { Caption } from 'components/Typography';
```

Comment:
All imports inside of the library should be full `components/Typography/Caption/Caption`